### ISSUE TITLE
Could not find a package configuration file provided by "the_Foundation"

### ISSUE THREAD

**Issue by tpapastylianou** [skyjake/lagrange#69]
I get the following error on trying to compile from source (v0.10.0 release) on linux (as per instructions).

    $ cmake ../ -DCMAKE_BUILD_TYPE=Release

    (bunch of normal messages here, then this error)

    CMake Error at CMakeLists.txt:38 (find_package):
      By not providing "Findthe_Foundation.cmake" in CMAKE_MODULE_PATH this
      project has asked CMake to find a package configuration file provided by
      "the_Foundation", but CMake did not find one.

      Could not find a package configuration file provided by "the_Foundation"
      with any of the following names:

        the_FoundationConfig.cmake
        the_foundation-config.cmake

      Add the installation prefix of "the_Foundation" to CMAKE_PREFIX_PATH or set
      "the_Foundation_DIR" to a directory containing one of the above files.  If
      "the_Foundation" provides a separate development package or SDK, be sure it
      has been installed.

I have no idea what this means, and google wasn't helpful. Any ideas?
(as the command implies, I made a 'build' directory inside the sources, but I get this message regardless of where I initiate the compilation)

**Comment by tpapastylianou** [comment]
Ok I figured it out. the_Foundation is a repository which the author hosts in his finish github account, but not github.com

I downloaded the sources for the_Foundation, replaced lib/the_Foundation empty folder with a symlink to the downloaded one from git, and then cmake completed successfully

Note to the author though, you should probably point this dependency out in the README ... I assume most people will reach your project through github.com and will not know what the_Foundation is about ...

**Comment by skyjake** [comment]
Thanks for the heads up. This GitHub repository is indeed just a mirror of the main one at https://git.skyjake.fi/skyjake/lagrange/. The full release notes on git.skyjake.fi mention which source tarball contains all the required source code.

**Comment by tpapastylianou** [comment]
Thanks for the tip. Just as a heads up, when I compiled using the git cloned version of the_Foundation, I kept getting Segfaults.

Compiling again from scratch from the 'inclusive' tarball you pointed out works great.

It's a beatiful browser, well done. I was intrigued to check it out after someone mentioned it on HackerNews (https://news.ycombinator.com/item?id=25226154) ... so ... possibly expect some more traffic!
